PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -1167,7 +1167,8 @@ private void ThisAddInShutdown(object sender, EventArgs e)
             PPMouse.StopHook();
             PPKeyboard.StopHook();
             PPCopy.StopHook();
-            PositionsPaneWpf.ClearAllEventHandlers();
+            //PositionsPaneWpf.ClearAllEventHandlers();
```

Comment:
@yuhongherald  Is there a reason for leaving this line as a comment?

Reply:
The clearing of event handlers have been moved into the destructor after changing the events from class level to object level.